index.html

```diff
@@ -206,24 +218,26 @@ <h4>Skills</h4>
       </ul>
     </section>
     <footer class="contact-group">
-      <div class="bgft">
-      <form action="index.html" method="post">
-      <div class="contact-me">
+    <div class="bgft">
+    <form action="index.html" method="post">
+      <div class="arrange-form">
+       <div class="contact-me">
         <h2 class="contact_align">Contact me</h2>
         <p class="pitch">
           If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
         </p>
-      </div>
-      <div class="organize_input">
-        <input type="text" name="username" placeholder="name" maxlength="30" required/>
+       </div>
+       <div class="organize_input">
+        <input type="text" class="username_text" name="username" placeholder="name" maxlength="30" required/>
         <input type="email" class="username_text" name="user_email" placeholder="email" required/>
         <textarea maxlength="500" name="textarea1" cols="30" rows="10" placeholder="write your text here" required></textarea>
+       </div>
       </div>
-      <div>
-        <button type="submit">Get it Touch</button>
+      <div class="desk-button">
+      <button type="submit">Get it Touch</button>
```

Comment:
- [ ] [OPTIONAL] Kindly stick to the Figma design as much as possible, you can add more margin for the Submit button and give it the proper colors when hovering and active state

![Screenshot_20220603_213312](https://user-images.githubusercontent.com/25227022/171948352-8bf42daa-8874-43a9-b078-f30046914468.png)